src/components/QRCode/HowTo.js

```diff
@@ -0,0 +1,45 @@
+import { h, Component } from 'preact'
+import classNames from 'classnames'
+
+import { localised } from '../../locales'
+import theme from '../Theme/style.css'
+import style from './style.css'
+
+class QRCodeHowTo extends Component {
+
+  constructor() {
+    super()
+    this.state = {
+      isExpanded: false
+    }
+  }
+
+  toggleHelpListVisibility = () => {
+    this.setState({ isExpanded: !this.state.isExpanded })
+  }
+
+  render() {
+    const { translate } = this.props
+    const { isExpanded } = this.state
+    return (
+      <div className={style.qrCodeHelp}>
+        <i className={style.qrCodeHelpIcon} />
+        <button
+          type="button"
+          aria-atomic="false"
+          aria-expanded={isExpanded}
+          className={classNames(theme.link, style.qrCodeHelpButton)}
+          onClick={this.toggleHelpListVisibility}>
+          {translate('cross_device.link.qr_code.how_to_label')}
+        </button>
+        <ul hidden={!isExpanded} className={style.qrCodeHelpList}>
+          <li data-onfido-qa="qrCodeHowToStep1" tabIndex="0">{translate('cross_device.link.qr_code.how_to_step_1')}</li>
```

Comment:
Why do you need tabIndex=0 here?

Reply:
Need it to bring the initially hidden list item elements into the tab order. Without it the list items weren't receiving focus when user continues tabbing through after activating the toggle to show it.